source/clear-linux/get-started/bare-metal-install/cgdisk-manual-install.rst

```diff
@@ -0,0 +1,303 @@
+figure 7 that there are 2 areas defined as free space. The first area at the
+top of the list, the 1007.0 KiB free space, is due to starting the previously
+defined EFI boot partition at sector 2048. This is discussed
+in `Rod Smith's Partitioning advice about alignment`_.
+
+#. Move your cursor to highlight the larger free space of 334.8 GiB at the
+   bottom of the partition list before you begin to create the Linux swap
+   partition. This is shown in figure 8:
+
+   .. figure:: figures/cgdisk-manual-install-8.png
+      :scale: 50 %
+      :alt: cgdisk - free space selection
+
+      Figure 8: :guilabel:`cgdisk - free space selection`
+
+#. To create the Linux swap partition, with the largest free space
+   highlighted, select the :guilabel:`[ New ]` button or press the :kbd:`N`
+   key and enter the following values for the Linux swap partition:
+
+   .. code-block:: console
+
+      First sector:  press :kbd:`Enter` to select the default value
+      Size in sectors:  4G
+      Hex code or GUID:  8200
+      Enter new partition name:  swap
+
+   Your :command:`cgdisk` partition list should now look like figure 9.
+
+   .. figure:: figures/cgdisk-manual-install-9.png
+      :scale: 50 %
+      :alt: cgdisk - swap partition defined
+
+      Figure 9: :guilabel:`cgdisk - swap partition defined`
+
+Create Linux filesystem partition
+*********************************
+
+The final partition that you will create is the Linux filesystem partition to
+be used as the root mount point for you |CL| installation.
+
+#. Highlight the largest free space entry at the bottom of the list and select
+   the :guilabel:`[ New ]` button or press the :kbd:`N` key and enter the
+   following values to create the Linux filesystem partition:
+
+   .. code-block:: console
+
+      First sector:  press :kbd:`Enter` to select the default value
+      Size in sectors:  press :kbd:`Enter` to select the default value, which
+                        will be the remainder of available space on the disk
+      Hex code or GUID:  8300
+      Enter new partition name:  root
+
+   With all the partitions now defined, you should see a list similar to what
+   is shown in figure 10:
+
+   .. figure:: figures/cgdisk-manual-install-10.png
+      :scale: 50 %
+      :alt: cgdisk - defined partitions
+
+      Figure 10: :guilabel:`cgdisk - defined partitions`
+
+#. If you are satisfied that the partition scheme is correct, you will need to
+   write this GPT to the hard drive. Select the :guilabel:`[ Write ]` button
+   or press the :kbd:`W` key and you will be prompted with:
+
+   .. code-block:: console
+
+      Are you sure you want to write the partition table to disk? (yes or no)
+
+#. Enter ``yes`` and press :kbd:`Enter` to write this data to the hard drive
+   and then select the :guilabel:`[ Quit ]` button or press :kbd:`Q` to exit
+   the :command:`cgdisk` utility and return to the |CL| manual installation

```

Comment:
This statement is false. The reader still needs to keep going before moving on to the manual installation. Please remove this reference.

Reply:
yes and no.  In this context yes, but the cgdisk program exits and returns to the installer application that launched it.  I've modified as requested